src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -56,22 +56,23 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
+  public List<Message> getMessages(String recipient) {
```

Comment:
Since there's now a chance for ambiguity about what this method is doing, let's go ahead and update the method name (maybe to something like `getMessagesByRecipient`?) and the javadoc comment.

Reply:
Thanks for the comments! I deleted the blank lines and changed the function name and javadoc